app/models.py

```diff
@@ -353,6 +353,36 @@ def __repr__(self):
         return '<MatomoDailyGetDatasetPageViewsSummary {}>'.format(self.id)
 
 
+class MatomoDailyGetPortalDownloadSummary(db.Model):
+    """
+    Provides Matomo daily download summary to store the statistics
+    on the different downloads via the portal.
+
+    Chosen variables from the endpoint are:
+    - url              = page URL
+    - label            = page label
+    - nb_hits          = number of views on this page
+    - nb_visits        = number of visits (30 min of inactivity considered a new visit)
+    - nb_uniq_visitors = number of unique visitors
+    - sum_time_spent   = total time spent on this page, in seconds
+    - segment          = segment with download URL
+    """
+    __tablename__ = 'matomo_daily_portal_download_summary'
+
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    date = db.Column(db.String(12))
+    url = db.Column(db.Text)
+    label = db.Column(db.String(256))
```

Comment:
why is url of type Text and not label or segment?

Reply:
the URL could be longer than 256 characters, which caused the insert statements to fail sometimes so I had to modify the type to Text instead of String. That was not an issue for the label or segment fields